### ISSUE TITLE
OpenML library upgrade

### ISSUE THREAD

**Issue by sebhrusen** [openml/automlbenchmark#62]

**Comment by PGijsbers** [comment]
Does this mean raising the dependency from 0.7.0 to 0.10x?

**Comment by sebhrusen** [comment]
last version I tried months ago was `0.9.0`, and there were some issues, so if `0.10.x` is available, we can try to switch directly to that one.

**Comment by franchuterivera** [comment]
Hello! Is there any plan to move to a new opeml version alongside a new sklearn version? 
I ask because there are some new metrics in sklearn it would be nice to enable (Balanced accuracy).
I tried 0.10.2 and did not see any issues while moving, but please let me know!

**Comment by sebhrusen** [comment]
Hey @franchuterivera, I'm definitely interested if you have a working branch (working with all current frameworks) with upgraded openml + sklearn.
Thanks!

**Comment by franchuterivera** [comment]
ok, I have been running smoothly using:
https://github.com/openml/automlbenchmark/pull/129  --> based on 0.10.2

There are some frameworks  I don't run (like constant predictor, etc) so let me try them out and make sure... 

But at least with 0.10.2 I don't see a problem with the majority of frameworks. I will confirm shortly...

**Comment by franchuterivera** [comment]
Please take a look at #129. I did not experience any problem while moving to the new openml benchmark version. Maybe some 0.9 problems that are now fixed.

All frameworks (except oboe) run smoothly. I need to debug (why oboe doesn't run) it on our end, but doesn't seem a problem with this transition.

Do notice that I am also fixing some typos I saw in the code and I am enabling balacc. I did not add it to the config but I would like to ask your opinion on that. Personally, i think balanced accuracy should be enabled. This means we have to tweak the frameworks for support on that. Let me know your thoughts
